Improved the Documentation structure (#1146)
pyansys/pyaedt: doc/source/API/CoreEdb.rst

EDB Editor
==========
An aedb database is a folder that contains the database representing any part of a PCB.

> It can be open and edited using ``Edb`` class.

Reviewer: ```suggestion
It can be opened and edited using ``Edb`` class.
```

.. currentmodule:: pyaedt

.. autosummary::
   :toctree: _autosummary

   Edb

.. code:: python

    from pyaedt import Edb

>     # this call return the Edb Class initialized on 2022 R1

Reviewer: ```suggestion
    # this call returns the Edb Class initialized on 2022 R1
```

   stackup.EdbStackup

.. code:: python

    from pyaedt import Edb
    edb = Edb(myedb, edbversion="2022.1")

>     # this call return the EdbHfss Class
>     comp = edb.core_hfss
> 
>     # this call return the Components Class
>     comp = edb.core_components
> 
>     # this call return the EdbSiwave Class
>     comp = edb.core_siwave
> 
>     # this call return the EdbPadstacks Class
>     comp = edb.core_padstack
> 
>     # this call return the EdbStackup Class
>     comp = edb.core_stackup

Reviewer: ```suggestion
    # this call returns the EdbHfss Class
    edb_hfss = edb.core_hfss

    # this call returns the Components Class
    edb_components = edb.core_components

    # this call returns the EdbSiwave Class
    edb_siwave = edb.core_siwave

    # this call returns the EdbPadstacks Class
    edb_padstack = edb.core_padstack

    # this call returns the EdbStackup Class
    edb_stackup = edb.core_stackup
```
